package.json

```
@@ -1,8 +1,12 @@
 {
   "name": "next-router-mock",
-  "version": "0.9.13",
+  "version": "0.11.0",
   "description": "Mock implementation of the Next.js Router",
   "main": "dist/index",
+  "exports": {
+    ".": "./dist/index.js",
+    "./navigation": "./dist/navigation/index.js"
+  },
```

There are a few more entries that should be added here, I'll work on making a list...

---

This ought to support the additional paths, like:
```
import "next-router-mock/async"
import "next-router-mock/navigation"
import "next-router-mock/MemoryRouterProvider"
import "next-router-mock/dynamic-routes"
```

```suggestion
  "exports": {
    ".": "./dist",
    "./*": "./dist/*",
    "./package.json": "./package.json"
  },
```